src/eureka/S5_lightcurve_fitting/fitters.py

```diff
@@ -860,15 +894,29 @@ def dynestyfitter(lc, model, meta, log, **kwargs):
         ind = [i for i in np.arange(len(freenames))
                if freenames[i][0:11] == "scatter_ppm"]
         for chan in range(len(ind)):
-            lc.unc_fit[chan*lc.time.size:(chan+1)*lc.time.size] = \
-                fit_params[ind[chan]] * 1e-6
+            if meta.multwhite:
+                trim1 = np.nansum(meta.mwhites_nexp[:chan])
```

Comment:
Given how many times I've seen these lines of code, I'm wondering if we should turn them into one or more routines, just in case we need to make changes down the road. Thoughts?

Reply:
Good point - 100% agree